### PR TITLE
Simpler coding (use SPLIT_KEYBOARD)

### PR DESCRIPTION
**Pull request by omkbd**
## Description
use SPLIT_KEYBOARD
delete rev1 (rev2→rev1)  because rev1is not sold

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* split code is old in ergodash.
* rev1 is prototype

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ergodash/rules.mk [hunk 1]
```diff
@@ -57,19 +26,18 @@ BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
...
 BACKLIGHT_ENABLE = no      # Enable keyboard backlight functionality
 MIDI_ENABLE = no            # MIDI controls
 AUDIO_ENABLE = no           # Audio output on port C6
 UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
-RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight. 
-SUBPROJECT_rev1 = yes
-USE_I2C = yes
+RGBLIGHT_ENABLE = no       # Enable WS2812 RGB underlight.
+USE_I2C = no
```
Comment (drashna) [inline keyboards/ergodash/rules.mk]: This doesn't actually do anything. 

```suggestion
```

Instead, the config.h should have it defined, if it's being used.

**Comment by yanfali** [comment]
What happened to rev2? And will this be confusing to users. Can you please add some information to the README as to what's happening here please. @omkbd

**Comment by omkbd** [comment]
> What happened to rev2? And will this be confusing to users. Can you please add some information to the README as to what's happening here please. @omkbd

The original Rev 1 was owned only by me. Therefore, Rev1 has been deleted. Since the current PCB is Rev1, I changed what was Rev2 to Rev1 in order to match the firmware version.

File: keyboards/ergodash/rules.mk (hunk 1 above)
Comment (omkbd) [reply keyboards/ergodash/rules.mk]: Thank you. I corrected.

**Comment by drashna** [comment]
Awesome, thanks!

**Comment by drashna** [comment]
You've opened this pull request from your fork's `master` branch. While this isn't a major issue now, later changes may result in merge conflicts if you try to file another PR because the commit history will have diverged between your master and QMK's. It is highly recommended for QMK development - regardless of what is being done or where - to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your master updated here:

[**Best Practices: Your fork's master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_best_practices?id=your-fork39s-master-update-often-commit-never)

If you need any help with this just ask.

**Comment by yourfate** [comment]
@omkbd, with this new update the RGB only works on the right half for me and a friend, we both have right hand master. I'm using this keymap, config, and rules.mk: https://gitlab.com/youRFate/keymaps/tree/master/ergodash/yf

**Pull request merged**